samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send a device message using the SendDeviceMessage endpoint. The d2c topic is used in non-bulk mode.
```

Comment:
it might be useful to add a link to https://docs.nrfcloud.com/Reference/DeviceMessages/

Reply:
Yes, that would certainly be useful! I will do so.